library/ajax/clinical_stats_and_lab_stats_by_demographics_report_ajax.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
- * library/ajax/clinical_stats_and_lab_stats_by_demographics.php: handles ajax calls from demographic lab/procedure reports
- * file adapted to present user activity log
+ * library/ajax/clinical_stats_and_lab_stats_by_demographics.php: handles ajax calls from demographic lab and procedure reports
+ *
```

Comment:
This is a reversion of changes that were already merged

Reply:
fixed